src/test/java/org/broadinstitute/hellbender/engine/dataflow/RefAPISourceUnitTest.java

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.hellbender.engine.dataflow;
+
+import com.google.cloud.dataflow.sdk.Pipeline;
+import com.google.cloud.dataflow.sdk.options.PipelineOptionsFactory;
+import com.google.cloud.dataflow.sdk.testing.TestPipeline;
+import com.google.cloud.genomics.dataflow.utils.GenomicsOptions;
+import org.broadinstitute.hellbender.engine.dataflow.datasources.RefAPIMetadata;
+import org.broadinstitute.hellbender.engine.dataflow.datasources.RefAPISource;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.reference.ReferenceBases;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+public class RefAPISourceUnitTest extends BaseTest {
+
+    private ReferenceBases queryReferenceAPI( final String referenceName, final SimpleInterval interval ) {
+        GenomicsOptions options = PipelineOptionsFactory.create().as(GenomicsOptions.class);
+        options.setApiKey(getDataflowTestApiKey());
+        options.setProject(getDataflowTestProject());
+
+        final Pipeline p = TestPipeline.create(options);
```

Comment:
(1) Not sure in what cases we'd want to use GATKTestPipeline, I don't expect any command line args (I think)
(2) It will depend on how Spark Dataflow handles the credentials. I don't think Tom & Company plan to do it this way anyway. They want to run on-site, which means getting the reference from some server or HDFS or something else.